ui/src/components/LandingPackageSIzes.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState, useEffect } from "react";
+import { PACKAGE_OPTIONS } from "../api/config";
+import axiosInstance from "../api/axiosInstance";
+import useMessage from "../hooks/messages";
+import BoxImage from "../images/box.png";
+import LetterImage from "../images/letter.png";
+import ParcelSizeCard from "./ParcelSizeCard";
+import DocumentSizeCard from "./DocumentSizeCard";
+import { Grid } from "@material-ui/core";
+
+function LandingPackageSizes() {
+  const { displayError } = useMessage();
+  const [parcelSizes, setParcelSizes] = useState(undefined);
+  const [letterSizes, setLetterSizes] = useState(undefined);
+
+  useEffect(() => {
+    (async function () {
+      try {
+        const response = await axiosInstance.get(PACKAGE_OPTIONS);
+        setParcelSizes(
+          response.data.filter((o) => o.packageType.title === "Package")
+        );
+        setLetterSizes(
+          response.data.filter((o) => o.packageType.title === "Document")
+        );
+      } catch (e) {
+        displayError("Failed to load package sizes");
+      }
+    })();
+  }, [displayError]);
+
+  return (
+    <>
+      <div style={{ marginTop: 120 }}>
+        <h2
+          style={{
+            display: "flex",
+            justifyContent: "center",
+            marginBottom: 60,
+          }}
+        >
+          Variety of Parcel Sizes
+        </h2>
+        <div className="form-wrapper">
+          <Grid container justify="space-between" spacing={5}>
+            {parcelSizes === undefined
+              ? ""
```

Comment:
No need for ternary operator here. You can write this as `!!parcelSizes && parcelSizes.map()`.